modules/core-services/services/member/member.go

```diff
@@ -602,7 +602,18 @@ func (m *Member) CheckPermission(userID string, scopeType apistructs.ScopeType,
 			return nil
 		}
 	}
+
+	admin, err := m.db.IsSysAdmin(userID)
+	if err != nil {
+		return err
+	}
+
 	switch scopeType {
+	case apistructs.SysScope:
+		// 系统管理员
+		if admin {
+			return nil
+		}
```

Comment:
why not check admin user's permission as same as other user?

I think you just query `m.db.GetMemberByScopeAndUserID(userID, scopeType, scopeID)` also work for admin?

thus `m.db.IsSysAdmin` may useless?

Reply:
`m.db.GetMemberByScopeAndUserID(userID, scopeType, scopeID)` will return all the members in this scope, and if you use this function, you need to do another traversal outside.  I think this old function is also quite good.